index.html

```diff
@@ -0,0 +1,234 @@
+                <div class="language-container">
+                    <div class="lang-heading"><img src="images/rhombus.png" alt="">
+                        <h3 class="iconic">Language</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Javascript</li>
+                        <li class="jss">Ruby</li>
+                        <li class="jss">Html</li>
+                        <li class="jss"> Css</li>
+                    </ul>
+                </div>
+                <div class="framecont">
+                    <div class="lang-heading"><img src="images/rectangle.png" alt="">
+                        <h3 class="iconic">Framework</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Bootstrap</li>
+                        <li class="jss">Ruby on Rails</li>
+                        <li class="jss">RSPec</li>
+                        <li class="jss">CapyBara</li>
+                        <li class="jss">Selenium</li>
+                    </ul>
+                </div>
+                <div class="skillscont">
+                    <div class="lang-heading"><img src="images/circle.png" alt="">
+                        <h3 class="iconic"></h3>Skills
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Codekit</li>
+                        <li class="jss">Github</li>
+                        <li class="jss">Codepen</li>
+                        <li class="jss"> Gitlab</li>
+                        <li class="jss"> Terminal</li>
+                    </ul>
+                </div>
+            </div>
+    </section>
+
+    <section class="reach">
+        <h1 class="int">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+            touch.</h1>
+        <form>
+            <label for="name"></label>
+            <input type="text" id="name" name="name" placeholder="Full name">
+
+            <label for="email"></label>
+            <input type="email" id="email" name="email" placeholder="Email address">
+
+            <label for="message"></label>
+            <textarea id="message" name="message" placeholder="Write something..."></textarea>
+
+            <input type="submit" value="Get in touch">
+        </form>
+
+        <div class="line2"></div>
+    </section>
+
+    <footer class="social-icons">
+        <ul id="icons">
+            <li class="github"><a href="#"><img src="./images/1.png" alt="github"></a></li>
+            <li class="linkedin"><a href="#"><img src="./images/2.png" alt="linkedln"></a></li>
+            <li class="angellist"><a href="#"><img src="./images/3.png" alt="angelist"></a></li>
+            <li class="twitter"><a href="#"><img src="./images/4.png" alt="twitter"></a></li>
+            <li class="facebook"><a href="#"><img src="./images/5.png" alt="facebook"></a></li>
+        </ul>
+    </footer>
```

Comment:
- [ ] Good job working on the project 👏  , but  I noticed that you add a part that should not be done in this milestone.  Kindly remove the contact section as it is a part of the next milestone 👍

Reply:
i have done that already

Reply:
what is the way forward